secure_message_v2/views/threads.py

```diff
@@ -64,3 +65,16 @@ def post_thread() -> Response:
         raise BadRequest(PAYLOAD_MALFORMED)
 
     return make_response(jsonify(thread.to_response_dict()), 201)
+
+
+@threads_bp.route("/<thread_id>", methods=["PATCH"])
+@jwt_authentication
+def patch_thread_by_id(thread_id: str) -> Response:
+    payload = request.get_json()
+    try:
+        thread = set_thread_attributes(thread_id, payload)
+        return make_response(jsonify(thread.to_response_dict()), 200)
```

Comment:
I kind of think you want all the thread back if updating, but we would probably need a use case to clarify whether a basic success is correct